dotfiles/.vimrc

```diff
@@ -1,18 +1,3 @@
-" -----------------
-" |   BOOTSTRAP   |
-" -----------------
-
-" first line, because it changes other options as side effect
-" most known setting as it allows usage of cursor keys
-set nocompatible
-
-" set runtime path dynamically based on user env home
-set runtimepath+=$DOTFILES_HOME/.vim
```

Comment:
can you write a few words on why you removed some lines that are not obviously related to the pathogen plugin (at least  it's not obvious to me)

Reply:
Because it's late and I shouldn't have. Nice catch :)